tests/sessions_tests/tests.py

```diff
@@ -198,6 +198,22 @@ def test_invalid_key(self):
             # session key; make sure that entry is manually deleted
             session.delete('1')
 
+    def test_session_key_empty_string_invalid(self):
+        """Falsey values (Such as an empty string) are rejected"""
+        self.session._session_key = ''
+        self.assertNotEqual(self.session.session_key, '')
```

Comment:
not sure I see the need to assertNotEqual when you assert the actual value in the next assertion?